buildlib/dockers-arm/ubuntu-release.Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG CUDA_VERSION=10.1
```

Comment:
Default looks obsolete, should be 11.x I suppose

Reply:
The default values in the Dockerfile are overridden by the values from docker-compose.yml.